cmd/minikube/cmd/root.go

```diff
@@ -70,8 +70,9 @@ var RootCmd = &cobra.Command{
 		}
 
 		shouldShowLibmachineLogs := viper.GetBool(showLibmachineLogs)
-
-		log.SetDebug(shouldShowLibmachineLogs)
+		if glog.V(3) {
+			log.SetDebug(shouldShowLibmachineLogs)
```

Comment:
Hmm, what about switching this back to log.SetDebug(true)?

Right now this is a little strange because if someone sets showLibmachineLogs to false in viper, we'll set it to false here even if glog is > v3